controllers/dellcsivolumegroupsnapshot_controller.go

```diff
@@ -110,6 +110,7 @@ func (r *DellCsiVolumeGroupSnapshotReconciler) Reconcile(ctx context.Context, re
 	}
 
 	if vg.Status.Status == common.EventStatusPending {
+		log.Info("&&&VG snapshotter status is pending&&&")
```

Comment:
Why need the &&& ? I would remove it

Reply:
i will remove it 
i had added it as a debug statement